localgov_forms.libraries.yml

```diff
@@ -6,6 +6,14 @@ localgov_forms_uk_address:
   dependencies:
     - webform/webform.composite
 
+localgov_forms_review_element:
+  version: VERSION
```

Comment:
Remove version, or else it will be cached until the version of Drupal core that is being used is updated.